CMakeLists.txt

```diff
@@ -52,10 +52,15 @@ option(EPROSIMA_BUILD "Activate internal building" OFF)
 option(EPROSIMA_BUILD_TESTS "Activate the building tests" OFF)
 option(THIRDPARTY "Activate the build of thirdparties" OFF)
 option(VERBOSE "Use verbose output" OFF)
+option(BUILD_INSTALLER "Activate the creation of a build to create windows installer" OFF)
```

Comment:
In other project we use EPROSIMA_INSTALLER. Can you rename this?

Reply:
Done, EPROSIMA_INSTALLER is not enable in Windows.